pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcFormat.scala

```diff
@@ -0,0 +1,173 @@
+package weco.pipeline.transformer.marc_common.transformers
+
+import weco.catalogue.internal_model.work.Format
+import weco.pipeline.transformer.marc_common.models.MarcRecord
+
+/** https://www.loc.gov/marc/bibliographic/bd008s.html "23 - Form of item
+  * (006/06)" The spec for field 008 contains a list of material specific
+  * details including the form of item, the code for which matches the value
+  * found in the 006 field at position 6.
+  */
+sealed trait FormOfItem
+object Marc008Field {
+  object FormOfItem {
+    case object Online extends FormOfItem
+  }
+}
+
+// Used to transform data from a MARC field where the data is positional
+// e.g. the leader and some control fields
+sealed trait MarcPositionalElement extends MarcDataTransformer {
+  val position: Int
+  val length: Int = 1
+}
+
+/** https://www.loc.gov/marc/bibliographic/bd006.html "Additional Material
+  * Characteristics" Marc 006 field is a control field that contains additional
+  * material characteristics, based on the form of material.
+  *
+  * For example, for continuing resources, the 006 field contains the form of
+  * item at position 6, but for visual materials it's at position 12.
+  *
+  * We infer the form of material based on values in the leader field.
+  *
+  * At present we only care about the form of item for books and continuing
+  * resources, in order to indentify e-books and e-journals (o in position 6).
```

Comment:
```suggestion
  * resources, in order to identify e-books and e-journals (o in position 6).
```
Whoops. I missed this typo first time round